### ISSUE TITLE
Address conflicting accessibility issues in H1 headings

### ISSUE THREAD

**Issue by victoriachanimal** [cds-snc/digital-canada-ca#2422]
The original website design features page titles / H1 headers in yellow inside the black bar at the top of each page: 

<img width="931" alt="Screen Shot 2021-07-21 at 12 47 15 PM" src="https://user-images.githubusercontent.com/39972942/126527579-b4f7de45-278a-4d31-981f-721f5df6366b.png">

In the redesign of certain Talent-related pages, including "Meet the team" and "Our values," we move the H1 headers to a white background based on WCAG guidelines that suggest grouping related elements with a single background colour can ensure the relationship and connection between the H1 and the rest of the content is preserved, and in order to address the [Info and relationship criterion](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html). 

<img width="922" alt="Screen Shot 2021-07-21 at 12 58 21 PM" src="https://user-images.githubusercontent.com/39972942/126528851-412f1ed7-93a8-4508-af5b-412ff9800baa.png">

**CONFLICT:** While the above redesign may fix one issue, it affects the header structure for redesigned pages, leaving them without H1s — creating an accessibility issue for screenreaders and failing WCAG's [Success Criterion 2.4.6 (2.4.6)](https://www.w3.org/TR/WCAG21/#headings-and-labels) and [2.4.2 (A requirement)](https://www.w3.org/TR/WCAG21/#page-titled). _Note: this is also captured in #2420._ 

As we still have more pages we will redesign, we will need to figure out how to align the redesign with the old framework, and a long-term plan to cut all base templates to fix these accessibility issues.
